src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+
+    @BeforeEach
+    void beforeEach() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_userCannotBeNull_NotOK() {
+        assertThrows(RuntimeException.class, () ->
+                registrationService.register(null));
+    }
+
+    @Test
+    void register_isUserHasNullParameter_NotOK() {

```

Comment:
In this case, only the case if age null is tested. Create two more tests for email and password